db/import.go

```diff
@@ -124,20 +124,7 @@ func (db *DatabaseCollectionWithUser) importDoc(ctx context.Context, docid strin
 	var newRev string
 	var alreadyImportedDoc *Document
 
-	mutationOptions := &sgbucket.MutateInOptions{}
-	if db.dataStore.IsSupported(sgbucket.BucketStoreFeaturePreserveExpiry) {
-		mutationOptions.PreserveExpiry = true
-	} else {
-		// Get the doc expiry if it wasn't passed in and preserve expiry is not supported
-		if expiry == nil {
-			getExpiry, getExpiryErr := db.dataStore.GetExpiry(ctx, docid)
-			if getExpiryErr != nil {
-				return nil, getExpiryErr
-			}
-			expiry = &getExpiry
-		}
-		existingDoc.Expiry = *expiry
-	}
+	mutationOptions := &sgbucket.MutateInOptions{PreserveExpiry: true}
```

Comment:
I think we still need to fetch the current expiry from the bucket if the Couchbase Server version we're using doesn't support PreserveExpiry?

Reply:
All versions of server 3.1.2 supports (CBS 7.0) support PreserveExpiry but I can leave this code in place, it was mostly for clarity here.